### PR TITLE
sepolicy for intel evs app

### PR DESCRIPTION
**Pull request by shivasku82**

### CODE REVIEW

**Review by neofang7 — APPROVED** [review]
LGTM

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]
- remove changes from line 34 to 51 and add only needed policies
- Add tests done to commit message and also in commit message add a Note: EVS intel app is customized version of default evs app to support virtio and ....

**Review by JeevakaPrabu — APPROVED** [review]

File: car/file_contexts [hunk 1]
```diff
@@ -3,3 +3,7 @@
 
 /vendor/bin/hw/android.hardware.broadcastradio@intel-service u:object_r:hal_broadcastradio_default_exec:s0
 /vendor/bin/hw/android.hardware.automotive.audiocontrol@1.0-service.intel u:object_r:hal_audiocontrol_default_exec:s0
+
+/system/bin/evs_intel_app                                             u:object_r:evs_intel_app_exec:s0
+/system/bin/evs_app_support_lib                                 u:object_r:evs_intel_app_exec:s0
```
Comment (ceiba1985) [inline car/file_contexts:8]: It's not proper to put Intel's binary to system partition. It will be lost after flashing the GSI image.

**Pull request merged**